### ISSUE TITLE
🐛 API | CreatedUTC 

### ISSUE THREAD

**Issue by tkapa** [SSWConsulting/SSW.Rewards.Mobile#749]
Cc: @GordonBeeming @matt-goldman @Anton-Polkanov 

Hi @tkapa 

### Describe the Bug

Since about late 2022, our Entities have not been Generated with a `CreatedUTC` Date.

As per my conversation with Gordon, this is likely due to missing Entity Configuration.

### To Reproduce
1. Take a look at the SQL DB (Staging or Prod)
2. See that Users + other Entities have a null/default CreatedUTC Date

### Tasks
- [ ] Investigate
- [ ] Fix

### Acceptance Criteria

- [ ] Meets Definition of Done - {{ LINK }}
- [ ] ...

### Environment
 - Device: [e.g. iPhone 12]

### Screenshots

Thanks!

**Comment by matt-goldman** [comment]
@tkapa I had a quick look at this and there's some weirdness going on. Firstly `User` inherits `BaseEntity` directly instead of `BaseAuditbaleEntity`. This is why it's not getting picked up by the interceptor. So changing this will require a bit of a refactor but also migrations and data migration.

But the weird thing is in the `RegisterProfile` command handler the `CreatedUtc` property is set directly. So no quick fix for this I'm afraid, it will need some investigation.

**Comment by tkapa** [comment]
Blocked - @matt-goldman, can this one be closed with #780?

**Comment by matt-goldman** [comment]
@tkapa I don't think this can be closed. #780 is a workaround, this issue still needs to be investigated. Also we need to verify whether #780 has worked (should be easy enough to check whether we can see the signups from DDD).

**Comment by AntPolkanov** [comment]
Might be relevant #827 and #782
